### PR TITLE
Add conll2002

### PR DESCRIPTION
**Pull request by lhoestq**
Adding the Conll2002 dataset for NER.

More info here : https://www.clips.uantwerpen.be/conll2002/ner/

### Checkbox

- [x] Create the dataset script `/datasets/my_dataset/my_dataset.py` using the template
- [x] Fill the `_DESCRIPTION` and `_CITATION` variables
- [x] Implement `_infos()`, `_split_generators()` and `_generate_examples()`
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is filled with the different configurations of the dataset and that the `BUILDER_CONFIG_CLASS` is specified if there is a custom config class.
- [x] Generate the metadata file `dataset_infos.json` for all configurations
- [x] Generate the dummy data `dummy_data.zip` files to have the dataset script tested and that they don't weigh too much (<50KB)
- [x] Add the dataset card `README.md` using the template : fill the tags and the various paragraphs
- [x] Both tests for the real data and the dummy data pass.

### CODE REVIEW

File: datasets/conll2002/README.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+|                            | Tain   | Valid | Test |
+| -----                      | ------ | ----- | ---- |
+| N. Examples (Spanish)      | 8324   | 1916  | 1518 |
+| N. Examples (Dutch)        | 15807  | 2896  | 5196 |
+
+## Dataset Creation
+
+### Curation Rationale
+
```
Comment (yjernite) [inline datasets/conll2002/README.md:138]:
```suggestion
[More Information Needed]

```

File: datasets/conll2002/README.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+#### Who are the source language producers?
+
+State whether the data was produced by humans or machine generated. Describe the people or systems who originally created the data.
+
+If available, include self-reported demographic or identity information for the source data creators, but avoid inferring this information. Instead state that this information is unknown. See [Larson 2017](https://www.aclweb.org/anthology/W17-1601.pdf) for using identity categories as a variables, particularly gender.
+
+Describe the conditions under which the data was created (for example, if the producers were crowdworkers, state what platform was used, or if the data was found, what website the data was found on). If compensation was provided, include that information here.
+
+Describe other people represented or mentioned in the data. Where possible, link to references for the information.
```
Comment (yjernite) [inline datasets/conll2002/README.md:157]:
```suggestion
[More Information Needed]
```

**Review by yjernite — CHANGES_REQUESTED** [review]
Looking good! A couple of paragraphs got squashed, I re-added the More Info Needed tag

**Review by yjernite — APPROVED** [review]

File: datasets/conll2002/README.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+- es:
+  - es
+- nl:
+  - nl
+licenses:
+- unknown
+multilinguality:
+- multilingual
+size_categories:
+- 10k<n<100K
```
Comment (lhoestq) [inline datasets/conll2002/README.md:16]:
```suggestion
- 10K<n<100K
```

**Pull request merged**